src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  const DAY_RENT = 40;
+  const BigDiscount = 50;
```

Comment:
The constant `BigDiscount` should be in all uppercase with underscores to follow JavaScript naming conventions for constants, as per checklist item 2. Consider renaming it to `LONG_TERM_DISCOUNT`.